evasquez26/angular4-startup
# S6 one 221 approve reject organization request

## Story / Task

[ONE-221](https://bixlabs.atlassian.net/browse/ONE-221)


## What this PR does?

Allows a super admin to approve, reject, see details and add comments of an organization creation request.


### Before 

We could approve or reject organization creation requests.

### After

Super global admins can approve and reject organization creation request.

### How to test / reproduce

- Login as any role.
- Navigate to schools -> new school
- Fill all the required info
- A ticket will be created, go to Ticket Management -> Organization
- Observe there's a new ticket with the information provided
- Login as a super global admin (evasquez@bixlabs.com)
- Navigate to Ticket Management -> Organization
- Observe that you can accept or reject the organization ticket
- Clicking accept should create the organization and set the corresponding user as a super admin of that organization (should be listed on School menu on that user) and it should close the ticket
- Clicking reject should close the ticket and allow the super admin to make a comment about the ticket

### TODOs

N/A

### Related PRs

N/A

## Migrations

NO

Thanks!
